src/test/java/homework13/BlackJackTest.java

```diff
@@ -0,0 +1,137 @@
+package homework13;
+
+import org.example.lesson14.blackjack.*;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class BlackJackTest {
+
+    @Test
+    @DisplayName("Check that hand isn't empty after adding a card")
+    void isNotEmptyHandTest() {
+        var hand = new BlackJackHand();
+        var deck = new Deck();
+        hand.addCard(deck.drawCard());
+        int sizeOfHand = hand.getCards().size();
+
+        assertEquals(1, sizeOfHand, "There are more than one item in the hand");
+    }
+
+    @Test
+    @DisplayName("Check that the card value is equal to the added card value")
+    void cardValueTest() {
+        var hand = new BlackJackHand();
+        Card card = new Card(Suit.DIAMONDS, Rank.JACK);
+        hand.addCard(card);
+
+        assertEquals(card, hand.getCards().get(0), "Card values are different");
+    }
+
+    @Test
+    @DisplayName("Check that the value biggest then 21 is busted")
+    void isBustedTest() {
+        int value;
+        boolean result = false;
+        do {
+            var hand = new BlackJackHand();
+            var deck = new Deck();
+            deck.shuffle();
+            hand.addCard(deck.drawCard());
+            hand.addCard(deck.drawCard());
+            hand.addCard(deck.drawCard());
+            value = hand.getValue();
+            if (value > 21) {
+                result = hand.isBusted();
+            }
+        } while (value <= 21);
+
+        assertTrue(result, "Value = " + value + ", and result of isBusted method = " + result);
+
+    }
+
+    @Test
+    @DisplayName("Check that 21 is blackjack")
+    void isBlackjackTest() {
+        int value;
+        boolean result = false;
+
+        do {
+            var hand = new BlackJackHand();
+            var deck = new Deck();
+            deck.shuffle();
+            hand.addCard(deck.drawCard());
+            hand.addCard(deck.drawCard());
+            hand.addCard(deck.drawCard());
+            value = hand.getValue();
+            if (value == 21) {
+                result = hand.isBlackJack();
+                break;
+            }
+        } while (value != 21);
```

Comment:
the same, why not like this?
```suggestion
        BlackJackHand blackJackHand = new BlackJackHand();
        blackJackHand.addCard(new Card(Suit.CLUBS, Rank.TEN));
        blackJackHand.addCard(new Card(Suit.HEARTS, Rank.ACE));
```